packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -427,6 +429,25 @@ Future<XcodeBuildResult> buildXcodeProject({
               return;
             }
           } else {
+            if (!globals.logger.isVerbose) {
+              if (line.contains('error:')) {
+                globals.printError(line);
+              } else if (line.contains('warning:')) {
+                globals.printWarning(line);
+                inWarningBlock = true;
+              } else if (inNoteBlock) {
+                globals.printWarning(line);
+                inNoteBlock = false;
+              } else if (inWarningBlock) {
+                if (line.startsWith(RegExp(r'\s+?note[:]'))) {
```

Comment:
```suggestion
                if (line.startsWith(RegExp(r'\s*?note[:]\s*?$'))) {
```

Notes can also be a single line. This regex shouldn't match if it is
```
note: hello world
```